doc/source/contribute.rst

```diff
@@ -0,0 +1,45 @@
+Contribute
+##########
+
+Thank you for your interest in contributing to PyWorkbench. Contributions for
+making the project better can include fixing bugs, adding new features, and
+improving the documentation.
+
+.. important::
+
+    This project adheres to the `Contributor Covenant Code of Conduct`_. By
+    participating, you agree to uphold this code of conduct.
+
+Start by selecting your role in the project:
```

Comment:
```suggestion
Overall guidance on contributing to a PyAnsys repository appears in
`Contribute <https://dev.docs.pyansys.com/how-to/contributing.html>`_
in the *PyAnsys developer's guide*. Ensure that you are thoroughly familiar
with this guide before attempting to contribute to PyWorkbench.

To contribute to PyWorkbench, start by selecting your role in the project:
```
Adding content to make this Contribute topic and the one for PyHeart consistent.